AI/pyneogame/Engine.py

```diff
@@ -180,7 +180,13 @@ def test():
         com_arrs = [np.array_equal(a, b) for x, y in zip(
             setup_1, setup_2) for a, b in zip(x, y)]
         # There is a chance that arrays will be similar just by chance
-        assert sum(com_arrs) < 3
```

Comment:
Add a 2nd expression here instead to print a message, see https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement.